### ISSUE TITLE
Special Attack Counter doesn't always register BGS special attacks

### ISSUE THREAD

**Issue by Nope1252** [runelite/runelite#15318]
### Type

Incorrect behavior

### Operating System

Windows

### OS Version

Windows 10

### Bug description

When in a party, BGS specs don't always register. This happens both locally and for party members. I haven't found any correlation to explain why it happens. Doesn't seem to happen with DWH specs. 

### Screenshots or videos

Filesize is too big for GH but here's a Streamable clip:
https://streamable.com/g81mcu

Apologies for the weird client layout, but you can tell the 4 clients in the bottom right clearly get an exp drop on BGS specs but the BGS counter doesn't go up.

### RuneLite version

RuneLite version: 1.8.26
Launcher version: 2.1.5

### Logs

[client.log](https://github.com/runelite/runelite/files/9086339/client.log)
[launcher.log](https://github.com/runelite/runelite/files/9086340/launcher.log)

**Comment by Nope1252** [comment]
I can get more clips or record with a different layout to make it more clear if needed, let me know.

**Comment by Alexsuperfly** [comment]
This probably is caused by one ticking the specs

**Comment by Nope1252** [comment]
> This probably is caused by one ticking the specs

Yep, that'll do it. Would it be possible to check what weapon is being equipped and update special counter accordingly?

**Comment by hex-agon** [comment]
1 ticking specs should be supported, do you have a video of it happening?

**Comment by Nope1252** [comment]
> 1 ticking specs should be supported, do you have a video of it happening?

https://streamable.com/g81mcu

Pay attention to the 4 clients in the bottom right corner.

**Comment by Nightfirecat** [comment]
@Nope1252 Is this still an issue since #16605 was merged?
